.github/workflows/lint.yml

```
@@ -32,13 +32,7 @@ jobs:
       id: ruff
       if: steps.changed-python-files.outputs.all_changed_files != '' && !cancelled()
       run: |
-        ruff check --config=pyproject.toml --show-fixes --output-format=github ${{ steps.changed-python-files.outputs.all_changed_files }}
-
-    - name: Black
-      id: black
-      if: steps.changed-python-files.outputs.all_changed_files != '' && !cancelled()
-      run: |
-        black --config=pyproject.toml --check --diff --verbose ${{ steps.changed-python-files.outputs.all_changed_files }}
+        ruff format --check --config=pyproject.toml ${{ steps.changed-python-files.outputs.all_changed_files }}
```

Since the whole project is formatted
```suggestion
        ruff format --check --config=pyproject.toml .
```